spec/rspec/support/spec/stderr_splitter_spec.rb

```diff
@@ -101,19 +101,19 @@
 
   # This is essentially what the `to_stderr_from_any_process` matcher attempts
   # to do in CaptureStreamToTempfile.
-  it 'is able to restore the stream from a cloned StdErrSplitter', :pending => RSpec::Support::Ruby.jruby? do
-    cloned = $stderr.clone
-    expect($stderr.to_io).not_to be_a(File)
+  it 'is able to restore the stream from a cloned StdErrSplitter' do
+    cloned = splitter.clone
+    expect(splitter.to_io).not_to be_a(File)
 
     tempfile = Tempfile.new("foo")
     begin
-      $stderr.reopen(tempfile)
-      expect($stderr.to_io).to be_a(File)
+      splitter.reopen(tempfile)
+      expect(splitter.to_io.path).to eq(tempfile.path)
     ensure
-      $stderr.reopen(cloned)
+      splitter.reopen(cloned)
       tempfile.close
       tempfile.unlink
     end
-    expect($stderr.to_io).not_to be_a(File)
+    expect(splitter.to_io.path).not_to eq tempfile.path
```

Comment:
While I think this is a valid test for the code in StderrSplitter, I don't think it exercises the case I was trying to rely on in https://github.com/rspec/rspec-expectations/pull/1460 anymore. Depending on whether we actually want to release that PR, that may be a good thing? I hadn't actually intended [this PR](https://github.com/rspec/rspec-support/pull/598) to be merged until we had consensus on that one.